back-end/routes/taskManagementApi.js

```diff
@@ -3,9 +3,9 @@ const router = express.Router();
 const TaskController = require('../controllers/TaskController');
 
 // Login route
-router.post('/create', TaskController.create);
-router.post('/read', TaskController.read);
-router.post('/update', TaskController.update);
-router.post('/delete', TaskController.delete);
+router.post('/tasks', TaskController.create); // create
+router.get('/tasks', TaskController.read); // read
+router.post('/tasks/:id', TaskController.update); // update
```

Comment:
in update feature , can we update by username, email ? or we just can only update by id ?

Reply:
This ID is the task ID, not user ID.

Each user will have multiple tasks and each task will have a unique task ID. So we need the task ID from the request to update a task in the db. What scenario that you want to update a task by username/email ?